apps/web/lib/auth/next-auth-custom-adapter.ts

```diff
@@ -0,0 +1,88 @@
+import { IdentityProvider } from "@prisma/client";
+import type { Adapter } from "next-auth/adapters";
+
+import { Account, Prisma, PrismaClient, User, VerificationToken } from "@calcom/prisma/client";
+
+import { identityProviderNameMap } from "@lib/auth";
+
+/** @return { import("next-auth/adapters").Adapter } */
```

Comment:
What does this do?

Reply:
it's just JSDoc markup

Reply:
Basically is typescript without typescript